builds/misc/images-mqtt.yaml

```diff
@@ -2,8 +2,7 @@ trigger:
   batch: true
   branches:
     include:
-      - iiot
-      - preview/iiot
```

Comment:
I think we don't need to run mqtt images from preview or release. Images only used for perf tests atm.